### DISCUSSION TITLE
沒有riscv32 command not found

### DISCUSSION THREAD

**Discussion by ChenChung107360206** [bol-edu/HLS-SOC-Discussions#122] [Q&A]
我在執行run_sim時遇到問題，想問有沒有人也有碰到以及如何解決的
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/43467000/c5de6970-e4ca-40da-8dd9-d7d50a223281)

**Comment by LingChiYang** [comment]
同學好

請參照 https://github.com/bol-edu/caravel-soc_fpga-lab 安裝tool chain
